services/web/server/src/simcore_service_webserver/director_v2/_handlers.py

```diff
@@ -29,8 +29,8 @@
 
 
 class RequestContext(BaseModel):
-    user_id: UserID = Field(..., alias=RQT_USERID_KEY)
-    product_name: str = Field(..., alias=RQ_PRODUCT_KEY)
+    user_id: UserID = Field(..., alias=RQT_USERID_KEY)  # type: ignore
+    product_name: str = Field(..., alias=RQ_PRODUCT_KEY)  # type: ignore
```

Comment:
why the type ignore here?

Reply:
error: Required dynamic aliases disallowed  [pydantic-alias] 
(I think type: ignore is acceptable here)

Reply:
Have you tried with Literal ?